Minichain_v0/consensus.py

```
@@ -0,0 +1,47 @@
+    Mine a new block using Proof-of-Work.
+    Includes coinbase transaction as mining reward.
+    Increments nonce until hash has required leading zeros.
+    """
+    index = prev_block.index + 1
+    prev_hash = prev_block.hash
+    timestamp = time.time()
+    nonce = 0
+    
+    # Add coinbase (mining reward) as first transaction
+    coinbase = create_coinbase_tx(miner_address, MINING_REWARD, index)
+    all_txs = [coinbase] + transactions
```

Ruff (0.15.1)

[warning] Consider `[coinbase, *transactions]` instead of concatenation

Replace with `[coinbase, *transactions]`

(RUF005)